db/db_compaction_test.cc

```diff
@@ -74,6 +74,43 @@ class CompactionStatsCollector : public EventListener {
   std::vector<std::atomic<int>> compaction_completed_;
 };
 
+class DeletionTriggeredCompactionWithMinFileSizeTestListener
+    : public EventListener {
+ public:
+  explicit DeletionTriggeredCompactionWithMinFileSizeTestListener(
+      uint64_t min_file_size)
+      : kMinFileSize_(min_file_size) {}
```

Comment:
`uint64_t kMinFileSize_; ` I don't think this is a constant variable anymore that qualifies for constant variable naming convention.